### DISCUSSION TITLE
评论区混进了1450，大家可一定要仔细甄别呀！

### DISCUSSION THREAD

**Discussion by Ventery** [WeNeedHome/SummaryOfLoanSuspension#892] [Ideas]
如题

**Comment by GitHubKlod** [comment]
1450是什么

**Reply by 378804441** [reply]
岛蛙，绿蛆，台巴子

**Reply by TALK1949** [reply]
稻草人

**Reply by fnnnnng** [reply]
甄别你妈逼 自己有问题还能怪的别人头上，你国的烂尾楼台湾建的？

**Reply by Ventery** [reply]
> 甄别你妈逼 自己有问题还能怪的别人头上，你国的烂尾楼台湾建的？

傻逼，下次买个旧号来。刚注册几天就来BB活像个沙口

**Reply by fnnnnng** [reply]
老子翻墙注册就是为了骂你这种奴才
新号影响你家的烂尾楼了？你家的烂尾楼关新号什么事？

**Reply by fangqiangqiang007** [reply]
> 老子翻墙注册就是为了骂你这种奴才
> 新号影响你家的烂尾楼了？你家的烂尾楼关新号什么事？

跟这种自己利益受损到这种程度还不忘维护主人颜面的奴才有什么好说的

**Reply by Ventery** [reply]
> 甄别你妈逼 自己有问题还能怪的别人头上，你国的烂尾楼台湾建的？

笑死，原来是墙内的蛆啊？鸣不公有本事上街去暴力对抗去啊？还专门溢出来在这无能狂怒，跟谁沆瀣一气不好非要跟1450，怕不是血汗厂里的真奴才😅

**Reply by fnnnnng** [reply]
> > 甄别你妈逼 自己有问题还能怪的别人头上，你国的烂尾楼台湾建的？
> 
> 笑死，原来是墙内的蛆啊？鸣不公有本事上街去暴力对抗去啊？还专门溢出来在这无能狂怒，跟谁沆瀣一气不好非要跟1450，怕不是血汗厂里的真奴才😅

你又赢了？
看看多少人骂你，你是怎么能做到这么自信的呢

**Reply by chen961** [reply]
神蛆大战

**Comment by jingzz1** [comment]
请问各位1450，你们省的烂尾楼会烂在民众手里吗？

**Reply by mehhss** [reply]
台湾国也有烂尾楼，全世界都有烂尾楼，不过基本能解决，中国这种不了了之是全世界独一档的，尤其是数量上

**Reply by blisee** [reply]
太好了，相关部门肯定能看见，一石二鸟

**Reply by ppinkiris** [reply]
台湾屁大点地方 烂能烂几栋 有可比性吗

**Reply by 17304495411** [reply]
> 台湾国也有烂尾楼，全世界都有烂尾楼，不过基本能解决，中国这种不了了之是全世界独一档的，尤其是数量上

你踏马的台湾也能算国？哪门子国？天国吗？

**Comment by C-Joey** [comment]
只知道1450？ 如何定义呢  必须是拿1450这个经费的  你怎么能识别对方拿了经费呢  大陆人一天天的老是在考虑这种无聊的事情  
你也要防大陆政府出钱的公关更加重要吧，毕竟他们有直接利害关系

**Reply by YangTianKundream** [reply]
1450網軍：用来讽刺偏袒民进党的网络水军。 1450后来演变为台湾的网军的代名词。 部分大陆网民也使用这个词语，他们声称存在台湾组织的专门用来抹黑大陆的1450网军，并将网络上疑似台湾水军的人称为1450

**Comment by yjeff112** [comment]
开始转移问题到国家矛盾，种族矛盾，你咋不把女权拉出来溜溜呢

**Reply by 378804441** [reply]
这有什么可转移矛盾的？矛盾就在这里摆着，我们就是不喜欢台巴子岛蛙，看他们在这逼逼赖赖而已

**Reply by yjeff112** [reply]
他们爱说让他们说呗 这里言论自由  至于他们说什么  不听就行了，管好自己别被带偏 不要去试图禁止别人发言

**Reply by 378804441** [reply]
所以看见岛蛙就骂就对了。绿脑怪们没一个好东西， 以为是在声援你，其实就是幸灾乐祸。这种绿蛆就该骂。言论自由么

**Reply by chenjiasu** [reply]
> 这有什么可转移矛盾的？矛盾就在这里摆着，我们就是不喜欢台巴子岛蛙，看他们在这逼逼赖赖而已

不要代表我

**Reply by 378804441** [reply]
你是个抖M没办法，贱嗖的喜欢被骂，我改的严谨一点。面的被一些斯德哥尔摩综合征患者抓把柄。

这有什么可转移矛盾的？矛盾就在这里摆着，我就是不喜欢台巴子岛蛙，看他们在这逼逼赖赖而已。

**Reply by ppinkiris** [reply]
台巴子蛙蛙先死几个妈再来发帖

**Reply by csu125008** [reply]
先解决主要矛盾呀

**Reply by chen961** [reply]
台湾佬说话跟个太监一样，前几年在台企干活，有个40多岁的人，发邮件，在里面写着”你帮帮人家嘛“，我他妈直接吐了，有个鸡巴都不至于说出这么娘娘腔的话出来

**Comment by John-zhan** [comment]
> 如题

请解决问题，别提这种恶心人的矛盾对立观，目的是解决问题，不是你这样的在这里上升到区域矛盾，非傻即坏

**Comment by shawn-yee** [comment]
@Ventery
推荐一篇文章给你
鲁迅《聪明人和傻子和奴才》

音频版本,几分钟可以听完
https://www.youtube.com/watch?v=zqNlmd18NUk

**Comment by Ventery** [comment]
github也该推出显示ip功能了，目测楼上这几位有一半都是1450......改成简体字？台湾的一些口语习惯是改不掉的

**Reply by 18750767761** [reply]
github可不归朝廷管辖

**Reply by John-zhan** [reply]
> github也该推出显示ip功能了，目测楼上这几位有一半都是1450......改成简体字？台湾的一些口语习惯是改不掉的

智障懂王

**Reply by KerryCodes** [reply]
应该推出拉黑功能，屏蔽你这种智障！

**Reply by Phineario** [reply]
说的太对了！
GitHub上面1450遍地走，尤其是这个仓库，简直就是1450聚集地！
建议楼主带头前去冲掉！支持楼主！
https://github.com/programthink/zhao

**Reply by fnnnnng** [reply]
你妈是怎么生出你这种傻逼的？反正你已经废了，能让你妈再生一个吗？

**Reply by shawn-yee** [reply]
github也玩定位ip，最好能跨省抓捕才符合国内自干五的想法，当太监当上瘾了。

**Reply by kongji123** [reply]
确实智障，认知为零

**Reply by chen961** [reply]
上面起码一大半都是台湾地区，能进github的都能算资深网民。看上面那几个人说话都不像经常上网的样子

**Reply by macleer** [reply]
好多绿头蛙啊

**Comment by ghost** [comment]
笑死，还以为这里是国内论坛呐

**Reply by chen961** [reply]
说话能不能不要带语气词行不？跟他妈太监一样，民进党也没有招太监这个岗位吧？

**Reply by macleer** [reply]
谁说没招？上面不就是吗

**Comment by Noorhotebbe** [comment]
傻杯，不解决问题搞对立？

**Comment by jiushi555** [comment]
懂了，大陆烂尾，台湾全责

**Reply by ewkyrtas** [reply]
Awesome

**Reply by dagoohan** [reply]
一定是xxx的陰謀！

**Reply by kongji123** [reply]
一定是美的的阴谋

**Reply by Cegazhos** [reply]
大陆烂大陆的 台湾乡巴佬来凑合啥呢 莱猪肉吃太饱了是吧

**Comment by John-zhan** [comment]
这里能看见这种不解决问题，却挑起矛盾的破坏分子，真心寒

**Reply by chenjiasu** [reply]
不要带节奏说这些幼稚的话好吧，大家都不是白莲花了。国内五毛猖獗指鹿为马，颠倒是非的事情不要太多。所以呢，你期待五毛不来这闹可能吗？

**Comment by yanwuguangjun** [comment]
很多1450 是 特地注册个账号上来嘴炮的，想在这件事上找优越感，求一时之爽的，大家一定要忽视他们。

**Reply by chenjiasu** [reply]
抱歉这里不是国内了，说别人1450的基本上都是五毛带节奏的

**Comment by yooq** [comment]
小粉红和1450来咬一咬，看哪种狗凶一点

**Comment by b4o** [comment]
正直的中国人都很务实，不像那些天天摇尾乞怜的狗和他的白人主子，每次改朝换代都是换汤不换药，坐在上面的还在上面，躺在地上的还在地上，都21世纪了，竟然法律上的种族平等都搞不彻底，都建国两百年了换了几代人了，还在天天为了自由而斗争，就这样他们18代以后还是一个鸟样，天天为自由斗争，在上面的还上面，在下面的还是在下面，狗还是狗

**Reply by KerryCodes** [reply]
sb

**Reply by b4o** [reply]
> sb

sb

**Reply by dagoohan** [reply]
什麼痰反？樓主你就不會駡點別的就複製粘貼人家罵你？

**Reply by hktxt** [reply]
我竟无言以对。

**Reply by victorchen276** [reply]
都是嫉妒你们中国太自由太民主太平等了。哈哈哈哈哈哈哈哈

**Comment by hiroandhero** [comment]
开放评论区就会被粉红冲

**Comment by TALK1949** [comment]
和自己幻想中的敌人战斗，就是这么刺激
不信去搜一搜“网评员招聘”，能搜到大量的招聘信息，却在哪里都找不到1450的招聘信息，如果诸位谁有，请给个链接，我去观摩一下😂

**Reply by dagoohan** [reply]
要是大家人均五十萬，那把大家全舉辦了那應該是夠還房貸了

**Comment by miaodi** [comment]
粪坑国国民是这样，活的跟蛆一样也不觉得自己恶臭。

**Comment by hadesII** [comment]


**Reply by chen961** [reply]
GITHUB受尼玛逼的限，又不是你妈的逼，有什么受限的？
明天就在你妈的逼上用C语言刻printf("Hello World");

**Comment by tyler110119** [comment]
1450万新台币就能有这么大的能量，我想知道每年内外宣多少亿资金，天天被骂的跟狗似的，没审核拉偏架动不动就评论翻车，不知道这些部门吃了多少回扣呢。

**Reply by volans19** [reply]
中共每年的维稳经费是1.4万亿，占总GDP的10%，是台湾的50万倍，但绝大部分经费没有落到5毛头上，99.9%都被领导拿去买房买车包小三生孩子了

**Reply by chen961** [reply]
上面就是最典型的1450，去年中国GOD114W亿，1.4W亿就1%左右，在这里说10%。拿着10年前的数据天天发，让你们领导更新下数据好不好？

**Comment by gongyongzhuanghu** [comment]
粉蛆遍地爬啊 污染自由之地

**Comment by frshgrass** [comment]
臭sb 真那么护主心切 你变卖家产赔给我们吧  我保证一辈子念你好 要不然我就是被卖又念别人好的sb

**Comment by allenaij** [comment]
总感觉有人在高级黑，故意往沼气池里面扔石头。

**Reply by xijinping0** [reply]
沼气池就是要多突开：

[![突开](https://img.youtube.com/vi/B8V4rrKuTkE/0.jpg)](https://youtu.be/B8V4rrKuTkE?t=113)

----

更多包蜜资讯：[xijinping0](https://github.com/xijinping0)

**Comment by shawn-yee** [comment]
1450又怎么样，你关注点应该是别人说的有没有道理，而不是别人背景，台湾几千万人都1450又怎么样，脑子长在你脖子上。

**Reply by TALK1949** [reply]
用否定消息源的方法来否定事实，这样就不需要论证消息是否属实，只需要不断地污名化消息源就可以了，宣传的老套路。

**Reply by chen961** [reply]
我最希望的就是明天房地产就崩溃，换人间。但是也不妨碍我看到台湾佬，我就要骂蛙蛙两句，草泥马的臭逼台湾佬

**Reply by fnnnnng** [reply]
> 我最希望的就是明天房地产就崩溃，换人间。但是也不妨碍我看到台湾佬，我就要骂蛙蛙两句，草泥马的臭逼台湾佬

现实唯唯诺诺，网上重拳出击

**Reply by Cegazhos** [reply]
> > 我最希望的就是明天房地产就崩溃，换人间。但是也不妨碍我看到台湾佬，我就要骂蛙蛙两句，草泥马的臭逼台湾佬
> 
> 现实唯唯诺诺，网上重拳出击

能不能反攻啊 别老缩在岛上好不好

**Reply by oceanzhang860411** [reply]
> > > 我最希望的就是明天房地产就崩溃，换人间。但是也不妨碍我看到台湾佬，我就要骂蛙蛙两句，草泥马的臭逼台湾佬
> > 
> > 
> > 现实唯唯诺诺，网上重拳出击
> 
> 能不能反攻啊 别老缩在岛上好不好

你想的美，人家全民醫療為什麼要給你一個網絡義和團享受？

**Reply by chen961** [reply]
全民茶叶蛋？

**Comment by xijinping0** [comment]
小王，把这里的1450整理出来，拉个清单。等二十大我连任之后，集中处理！

----

推荐阅读：[《习近平与他的情人们》](https://xijinping0.github.io/lovers/docs/toc)

**Comment by Cegazhos** [comment]
没想到台湾乡巴佬这么多，自己家的莱猪吃太饱了 闲着没事干去关心岛外的烂尾
不知道的还以为乡巴佬进城买房了

**Comment by TaiWanIsPartOfChina** [comment]
![image](https://user-images.githubusercontent.com/86300347/185309538-440192bb-8873-4e83-88d8-55893c80104b.png)
台湾人よ、パイナップルを食べに来て
